react/components/atoms/text-input/text-input.js

```diff
@@ -0,0 +1,90 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, Text, TextInput, View } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin, baseStyles } from "../../../util";
+
+export class TextInputRipe extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            title: PropTypes.string,
+            placeholder: PropTypes.string,
+            value: PropTypes.string,
+            hasBorder: PropTypes.bool,
+            borderColor: PropTypes.string,
+            onValueUpdate: PropTypes.func
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            placeholder: "Insert value here",

```

Comment:
In the design, the placeholder has the name of the field, for example "Name". Maybe default to this.props.title and then default to this if not defined